ledger/sandbox-common/src/main/scala/platform/sandbox/cli/CommonCli.scala

```diff
@@ -33,14 +33,6 @@ import scala.util.Try
 // instead.
 class CommonCli(name: LedgerName) {
 
-  private implicit val clientAuthRead: Read[ClientAuth] = Read.reads {
-    case "none" => ClientAuth.NONE
-    case "optional" => ClientAuth.OPTIONAL
-    case "require" => ClientAuth.REQUIRE
-    case _ =>
-      throw new InvalidConfigException(s"""Must be one of "none", "optional", or "require".""")
-  }
-
```

Comment:
Is this not used anywhere?

Reply:
It is used in the `CommonCli` but there's an import: `import com.daml.platform.configuration.Readers._`. I've just moved that implicit to `Readers`.

Reply:
Excellent. Thanks.